style.css

```diff
@@ -0,0 +1,154 @@
+body {
+    background-color: #4A5AC0;
+}
+
+.presentation-header {
+    width: 100%;
+    height: 50px;
+    display: block;
+    background-color: rgb(246, 191, 8);
+    margin: 100px solid rgb(246, 191, 8);
+    
+    position: fixed;
+    left: 0;
+    top: 0;
+
+    color: white;
+    font-size: 25px;
+    text-shadow: 0 0 5px grey;
+}
+
+.header-words {
+    position: relative;
+    top: 10px;
+    left: 30px;
+}
+
+.slide  {
+    position: relative;
+    top: 50px;
+}
+
+.slide-title {
+    width: 70%;
+    font-size: 70px;
+    font-weight: bold;
+    position: relative;
+    top: 350px;
+    left: 250px;
```

Comment:
Pixel positioning usually causes issues with responsive layouts... or even layouts with different browsers / machines. Look into other ways of centering something vertically? E.g. top: 50%, then transform: translate(??? ???)?